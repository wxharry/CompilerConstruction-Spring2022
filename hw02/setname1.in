setname Alice
Yesterday I met name. name lives nearby.
setname Bob
Today I met name. name lives nearby too.