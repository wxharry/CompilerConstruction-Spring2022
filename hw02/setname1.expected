
Yesterday I met Alice. Alice lives nearby.

Today I met Bob. Bob lives nearby too.